### ISSUE TITLE
Unrecognised HE mesh

### ISSUE THREAD

**Issue by matvitali** [gmsafir/gmsafir#36]
Hello,
I am a civil engineer who is trying out the demo version of SAFIR to see if it might be of interest to purchase it for FSE purposes of the company I work for.

I tried meshing a commercial profile section (HEB400) as indicated in the youtube tutorial at [this link](https://www.youtube.com/watch?v=d0jsfSNwgv8&t=361s). I also try to build the section perimeter one edge a time, counterclockwise.
In gmsafir I get the desired mesh but the thermal analysis in SAFIR (demo version) reports errors (Jacobian < 0).
![image](https://user-images.githubusercontent.com/116348740/197165817-fefdab99-457b-4d17-b796-d035c043ebec.png)
In fact in Diamond I see that the mesh has not been read correctly.
![image](https://user-images.githubusercontent.com/116348740/197165921-34fef4c8-c2a6-4313-981b-0da907e4f391.png)
By generating a more simplified H-section instead (H), I do not detect any errors in the analysis.
I attach here the generated files and kindly ask you what the problem might be.
Thank you in advance.
Matteo

[HEB400.g4s.txt](https://github.com/gmsafir/gmsafir/files/9837811/HEB400.g4s.txt)
[HEB400.geo.txt](https://github.com/gmsafir/gmsafir/files/9837812/HEB400.geo.txt)
[HEB400.IN.txt](https://github.com/gmsafir/gmsafir/files/9837813/HEB400.IN.txt)
[HEB400.OUT.txt](https://github.com/gmsafir/gmsafir/files/9837814/HEB400.OUT.txt)
[HEB400.TEM.txt](https://github.com/gmsafir/gmsafir/files/9837815/HEB400.TEM.txt)
[HEB400.XML.txt](https://github.com/gmsafir/gmsafir/files/9837816/HEB400.XML.txt)
[H.g4s.txt](https://github.com/gmsafir/gmsafir/files/9837836/H.g4s.txt)
[H.geo.txt](https://github.com/gmsafir/gmsafir/files/9837837/H.geo.txt)
[H.IN.txt](https://github.com/gmsafir/gmsafir/files/9837838/H.IN.txt)
[H.OUT.txt](https://github.com/gmsafir/gmsafir/files/9837839/H.OUT.txt)
[H.TEM.txt](https://github.com/gmsafir/gmsafir/files/9837840/H.TEM.txt)
[H.XML.txt](https://github.com/gmsafir/gmsafir/files/9837841/H.XML.txt)

**Comment by gmsafir** [comment]
thx Wojciech for your kind help

**Comment by matvitali** [comment]
Thanks a lot Wojciech for your reply.
Following your list I add some comments that could be useful:

1. Ok I checked the counter clockwise line selection for surface generation.
2. Seems that the "coherence" command could solve the problem of duplicate entities, without the need to delete them manually. After "coherence" I see no overlapping point labels. The issue was mainly due to the "external" node that I used to build circular arc for fillets: I manually deleted only that one.

If you haven't other useful comments I can close with thanking you!

Best regards,
Matteo
